src/components/Testimonials/Testimonials.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+
+import React from 'react'
+import withStyles from 'isomorphic-style-loader/lib/withStyles'
+import Slider from 'react-slick'
+
+import s from './Testimonials.css'
+
+type Props = {
+  testimonials?: Array<string>,
+}
+
+const Testimonials = ({ testimonials }: Props) => {
+  const settings = {
+    dots: false,
+    infinite: false,
+    speed: 500,
+    slidesToShow: 3.5,
+    slidesToScroll: 1,
+    responsive: [
+      {
+        breakpoint: 480,
+        settings: {
+          slidesToShow: 1,
+        },
+      },
+      {
+        breakpoint: 768,
+        settings: {
+          slidesToShow: 2,
+        },
+      },
+    ],
+    accessibility: true,
+    adaptiveHeight: false,
+  }
+
+  return (
+    <div className="container">
+      <Slider {...settings}>
+        {testimonials.length && testimonials.map((item, key) => (
```

Comment:
no need to check for length. If there are no testimonials, the `.map` won't do anything